app/assets/stylesheets/refinery/stories.scss

```diff
@@ -1,5 +1,44 @@
 .StoriesPage {
 
+  .weigh-in__header {
+    display: flex;
+    background-color: $color_bg-lightest;
+    background-image: url('group-2111.png');
+    padding: 14.18rem 19.5rem 14.18rem 37.43rem;
+    
+    @media (max-width: 1600px){
+      padding: 14.18rem 19.5rem 14.18rem 14.43rem;
+    }
+
+    @media (max-width: 1125px){
+      padding: 14.18rem 19.5rem 14.18rem 10.43rem;
+    }
+
+    @media (max-width: 933px){
+      background-image: none;
+      padding: 6.25rem 3rem 3rem 1.56rem;
+    }
+    
+    &--content {
+      display: 1;
+    }
+    
+    &--title {
+      color: #111436;
+      font-size: 2.125rem;
```

Comment:
I've generally been defining font sizes and colors over at https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/global/_vars.scss#L8-L16 and integrating these into the CSS file. I think you can potentially create vars and add these if they are new, but it might also be worth a discussion with Kit that each of these font sizes is different and see if it makes sense to standardize them a bit more.